packages/kit/src/exports/public.d.ts

```diff
@@ -35,6 +35,16 @@ export interface Adapter {
 	 * @param builder An object provided by SvelteKit that contains methods for adapting the app
 	 */
 	adapt(builder: Builder): MaybePromise<void>;
+	/**
+	 * Functions called during dev and build to determine whether specific features will work in production with this adapter
+	 */
+	supports?: {
```

Comment:
should we add something about this to https://kit.svelte.dev/docs/writing-adapters ?  our docs there are pretty limited, but could keep us from falling further behind

Reply:
done